app/Http/Controllers/Api/ConsumablesController.php

```diff
@@ -199,6 +199,8 @@ public function destroy($id)
     /**
     * Returns a JSON response containing details on the users associated with this consumable.
     *
+    * @todo DEPRICATE IN NEXT MAJOR RELEASE - /checkedout should be used instead
```

Comment:
(sp: DEPRECATE) 

But also I don’t know that this is the right method - this, as I understand it, is for deleting a consumable - whereas a `/checkedout` method call would be something else, right?